package.json

```diff
@@ -1,17 +1,18 @@
 {
-  "name": "@pnext/three-loader",
+  "name": "@spelman7/three-loader",
   "private": false,
-  "version": "0.1.1",
+  "version": "0.1.8",
```

Comment:
We shouldn't modify this sutff `package.json` since we can do that as part of a release. But I am ok with adding your name to the contributors.